src/map/script.c

```diff
@@ -15815,6 +15815,56 @@ static BUILDIN(specialeffect)
 	return true;
 }
 
+/*==========================================
+ * Special effects with num [4144]
+ *------------------------------------------*/
+static BUILDIN(specialeffectnum)
+{
```

Comment:
Maybe add something like this to prevent misuse of the command.
```
#if PACKETVER < 20191127
	ShowError("buildin_specialeffectnum: %s needs at least PACKETVER 20191127. You are using %d.", script->getfuncname(st), PACKETVER);
	st->state = END;
	return false;
#endif
```

Reply:
i think terminate script if something not supported is bad idea. because this is only effect, better ignore it as now

Reply:
At least print a warning, to notify the user that something is wrong.

Reply:
about error. here exists clients older than 2006?
because this packet not supported only in 2006 versions older than 2006

Reply:
I just noticed your PACKETVER switch in `clif_specialeffect_value`. Just nevermind my change request...